packages/lib/services/e2ee/types.ts

```diff
@@ -25,3 +25,14 @@ export interface RSA {
 	publicKey(rsaKeyPair: RSAKeyPair): string;
 	privateKey(rsaKeyPair: RSAKeyPair): string;
 }
+
+export interface KryptoInterface {
```

Comment:
Please name it "Crypto". We know from the type that it's an interface so it's not necessary to append this.

Reply:
OK